dandi/tests/test_metadata.py

```diff
@@ -359,3 +360,283 @@ def test_species():
         "schemaKey": "SpeciesType",
         "name": "Drosophila suzukii",
     }
+
+
+@pytest.mark.parametrize(
+    "ndtypes,asset_dict",
+    [
+        (
+            ["ElectricalSeries"],
+            {
+                "approach": ["electrophysiological approach"],
+                "measurementTechnique": [
+                    "multi electrode extracellular electrophysiology recording technique"
+                ],
+                "variableMeasured": ["ElectricalSeries"],
```

Comment:
What's the point of these dict values being lists if they're all just one element long?

Reply:
they could be extended in the future. i most realistic settings they will be more than one element long.